tpm2/Makefile

```diff
@@ -23,36 +23,36 @@ LLIBS += -ltss2-rc#                      TPM 2.0 Return Code Utilities
 LLIBS += -lssl#                          OpenSSL
 LLIBS += -lcrypto#                       libcrypto
 LLIBS += -lkmip#                         libkmip
-LLIBS += -lkmythlogger#                  kmyth logging utility shared library
+LLIBS += -lkmyth#                        kmyth shared library
 
 # Specify compiler flags
 CFLAGS = -Wall#                          enable all warnings
 CFLAGS += -c#                            compile, but do not link
 CFLAGS += ${DEBUG}#                      debugging options (above)
 CFLAGS += -D_GNU_SOURCE#                 GNU/LINUX platform
+CFLAGS += -fPIC#                         Generate position independent code
 
 KMYTH_CFLAGS = $(CFLAGS)
 KMYTH_CFLAGS += -I${LOGGER_DIR}/include#    kmyth logging utility header files
 
 # Specify flags for the SO build of the logger
 LOGGER_CFLAGS = $(CFLAGS)
-LOGGER_CFLAGS += -fPIC#                     Generate position independent code
 
 SOFLAGS = -shared#                          compile/link shared library
 
 # Specify linker flags
-LFLAGS += -L${LOGGER_DIR}/lib#          libkmythlogger
-LFLAGS += -Wl,-rpath=${LOGGER_DIR}/lib# runtime path for libkmythlogger.so
+LFLAGS += -Llib#                            libkmyth
+LFLAGS += -Wl,-rpath=lib#                   runtime path for libkmyth.so
```

Comment:
Again, I'm not sure that this really adds anything, but, if we moved the directory variables up, we could use -L$(LIBDIR) and -rpath=$(LIBDIR) to keep it consistent if that is ever changed for some reason.

Reply:
I think if someone is modifying this for a local build and test, it should be an intentional edit to that part of the Makefile.

Reply:
I agree that hard-coding the local build and test configuration is OK. I was just thinking that since we made a variable to parameterize other local build environment actions, we might use it here as well. I'm OK, though, with leaving it as is.